.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+# Overview
+
+### What is the feature/fix?
+
+Please summarize the feature or fix.
+
+### What is the Solution?
+
+Summarize what you changed.
+
+### What areas of the application does this impact?
+
+List impacted areas.
+
+# Testing
+
+### Performance Test Results
+
+Please put in details of performance results from testing in WL
```

Comment:
If we are uncomfortable putting performance results, then we can add this as a checklist instead. Please LMK what you think